adserver/analyzer/backends/naive.py

```diff
@@ -23,6 +23,18 @@ class NaiveKeywordAnalyzerBackend(BaseAnalyzerBackend):
         "body",
     )
 
+    # CSS selectors of content that should be ignored for the purpose of analysis
```

Comment:
This is the same logic that was in the training import code, just ported to the backend, right?

Reply:
Correct, ported to the analyzer.